engines/suppliers/app/collections/suppliers/orders_collection/to_group.rb

```diff
@@ -1,3 +1,5 @@
+require 'suppliers/orders_collection'
```

Comment:
It's because of the way OrdersCollection is implemented. Read the commit message for more details.